SPECS/kernel/config_aarch64

```diff
@@ -1007,7 +1007,6 @@ CONFIG_ZPOOL=y
 CONFIG_ZBUD=y
 CONFIG_Z3FOLD=m
 CONFIG_ZSMALLOC=y
-CONFIG_PGTABLE_MAPPING=y
```

Comment:
Maybe also mention that in the changelog to make it easier to track down.

Reply:
Updated PR description. Not sure if it is needed in the changelog entry itself.